src/triage/models/assertion.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+from django.db import models
+
+
+class AssertionsPerPackage(models.Model):
+    """All assertions of a Package"""
+
+    total_assertions = models.PositiveIntegerField()
+    package_name = models.CharField(max_length=100)
+    package_uuid = models.UUIDField(unique=True)
+    url = models.URLField(max_length=150, default=None)
+
+    def __str__(self):
+        return f"Assertion Summary for {self.package_name}"
+
+    def save(self, *args, **kwargs):
+        if self.package_uuid and not self.url:
+            # Generate the URL based on package_uuid and fixed URL pattern
+            self.url = f"https://oafdev1.westus2.cloudapp.azure.com/assertions/show?subject_uuid={self.package_uuid}"
```

Comment:
The assertion URL, can we store this into an environment variable? 

This may not always be the link, so with an environment variable, we'd only have to change it once and not multiple times within the code.

Reply:
Good point! Thanks for the advice